ow_lander/src/ow_lander/server.py

```diff
@@ -138,6 +138,14 @@ def __on_action_called(self, goal):
                    "This should never happen!")
       return
     rospy.loginfo(f"{self.name} action started")
+    # reset execution fault flag by sending a PENDING state
```

Comment:
Is the rest of this paragraph consistent with this first sentence, which says PENDING state?

Reply:
Whoops. I originally planned to reset it with a PENDING state. It does not ultimately matter given how ActionGoalStates works. I

Good catch. This is fixed.